pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -12,10 +12,10 @@ $border-radius: 4px;
 // Start styles
 html {
   scroll-behavior: smooth;
+  font-size: 100%;
```

Comment:
This is where I overrode the 10px html font-size — now it's using the default browser font-size which is 16px. ([previous comment](https://github.com/code-dot-org/code-dot-org/pull/52239#discussion_r1222104425))